Sleep Tracker/Sleep Tracker/View/SleepSetupView/HelpersView/RoundedAlarmView.swift

```diff
@@ -0,0 +1,28 @@
+import SwiftUI
+
+/// A custom button for the alarm clock which, when clicked, change its opacity property with animation.
+struct RoundedAlarmView: View {
+
+    /// A boolean value that indicates whether animation is enabled.
+    /// When `true` button has no transparency and when `false` button has transparency.
+    @Binding var animate: Bool
+
+    var body: some View {
+        Image.AppImage.alarmSymbolWhite
+            .resizable()
+            .frame(width: 20, height: 20)
+            .background {
+                RoundedRectangle(cornerRadius: 30)
+                    .fill(.purple)
+                    .frame(width: 70, height: 40)
+            }
+            .opacity(animate ? 1.0 : 0.5)
+            .zIndex(1)
```

Comment:
Can we simplify the code:

```suggestion
        Image.AppImage.alarmSymbolWhite
            .padding(EdgeInsets(top: 10, leading: 25, bottom: 10, trailing: 25))
            .background(.purple)
            .cornerRadius(30)
            .opacity(animate ? 1.0 : 0.5)
```

?

Reply:
I agree
fixed